src/sentry/static/sentry/app/utils/profiler.jsx

```diff
@@ -1,56 +1,83 @@
-import PropTypes from 'prop-types';
```

Comment:
@billyvg Any reason we aren't making this a TS file?

Reply:
ya it's still a wip